<p><b>[INFO]</b>&nbsp;GET Request : url=https://gateway.marvel.com/v1/public/characters <br> path_url=/v1/public/characters <br> headers={'User-Agent': 'python-requests/2.26.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'ts': '1631150721003', 'apikey': 'e9e22f92431df9f73920f281caabb68c', 'hash': 'f0b3298fa4957047893445f8d19881b8'} <br> body=None <br> </p><p><b>[INFO]</b>&nbsp;GET Response : url=https://gateway.marvel.com/v1/public/characters <br> status=409, reason=Conflict <br> headers={'Content-Encoding': 'gzip', 'Content-Type': 'application/json; charset=utf-8', 'Date': 'Thu, 09 Sep 2021 15:07:56 GMT', 'Connection': 'keep-alive', 'Transfer-Encoding': 'chunked'} <br> body={"code":"MissingParameter","message":"You must provide a user key."} <br> </p><p style="color: red"><b>[FAIL]</b>&nbsp;HTTPError: 409 Client Error: Conflict for url: https://gateway.marvel.com/v1/public/characters</p>